test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo_test_helper
```

Comment:
Remove this commit and move this change to the migration commit. There should be only one commit addressing migration changes.